vue-app/src/utils/localStorage.ts

```diff
@@ -0,0 +1,26 @@
+import pkg from '@/../package.json'
+
+function lsGetKey(key: string) {
+  return `${pkg.name}.${key}`
```

Comment:
Could you explain why we use `pkg.name` - just to give our keys a namespace?

Reply:
Yes, this was just to ensure a unique namespace